components/MarketStats/DaoStats.tsx

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react'
+import { Box } from '@zoralabs/zord'
+import { roundFourDecimals, numberFormatter } from '@shared'
+import { useAuctionVolume, useAggregate } from 'hooks'
+import { returnDao } from 'constants/collection-addresses'
+import { StatBlock } from './StatBlock'
+
+export function DaoStats({ contractAddress }: { contractAddress: string }) {
+  const { aggregate } = useAggregate(contractAddress)
+  const dao = returnDao(contractAddress)
+  /* @ts-ignore */
+  const { data } = useAuctionVolume(contractAddress, `${dao?.marketType}_SALE`)
+
+  const auctionVolume = useMemo(() => {
+    if (data?.chainTokenPrice) {
+      return `${numberFormatter(roundFourDecimals(data?.chainTokenPrice as number))} ETH`
+    } else {
+      return '...'
+    }
+  }, [data, data?.chainTokenPrice])
+
+  const secondaryVolume = useMemo(() => {
```

Comment:
With deep traversals like this I find it clearer to comprehend the logical flow by pulling the traversed result into a const with a name that's parsed easily at a glance, like:

```
const aggregateTokenPrice = aggregate?.aggregateStat?.salesVolume?.chainTokenPrice
```
so we then have

```if (data?.chainTokenPrice && aggregateTokenPrice)```

vs.

```if (data?.chainTokenPrice && aggregate?.aggregateStat?.salesVolume?.chainTokenPrice)```

Reply:
Cool - giving this a shot.